2016-01-06/swagger/api_v2.0.json

```diff
@@ -48,6 +48,15 @@
     "/{bucketName}?uploads": {
       "$ref": "./bucket_uploads.json"
     },
+    "/{bucketName}?Lifecycle": {
```

Comment:
No upper case query in API.

Reply:
No upper case query in API.

Reply:
We always use `PUT /?lifecycle` for API as described in https://docs.qingcloud.com/qingstor/api/bucket/lifecycle/